clusterloader2/testing/density/pod-pod-anti-affinity/overrides.yaml

```diff
@@ -0,0 +1,7 @@
+SATURATION_DEPLOYMENT_SPEC: pod-pod-anti-affinity/deployment.yaml
+LATENCY_DEPLOYMENT_SPEC: pod-pod-anti-affinity/deployment.yaml
+
+# The default CL2_SCHEDULER_THROUGHPUT_THRESHOLD value of 90 is too high for
+# pod-pod anti-affinity. Use a low threshold until a good baseline can be
+# identified and then increase the threshold incrementally.
+CL2_SCHEDULER_THROUGHPUT_THRESHOLD: 40
```

Comment:
is this for an alert? if so, please clarify that in the comment and I think we should keep it at 90 unless this will be executed for older versions.

Reply:
oops, this is an artifact from my testing when the throughput was below 50 for unrelated reasons. removing.